bin/analyze_0l_2tau.cc

```diff
@@ -1661,6 +1741,25 @@ int main(int argc, char* argv[])
           );
           }
         }
+
+	if(! Weight_ktScan.empty()  )
+	{
+	  for(const auto & kv: tH_weight_map)
+	  {
+	    for(std::size_t scanIdx = 0; scanIdx < Weight_ktScan.size(); ++scanIdx)
+	    {
+	      double evtWeight0 = evtWeight * Weight_ktScan[scanIdx] / HHWeight;
```

Comment:
Change to:
```cpp
double evtWeight0 = kv.second * Weight_ktScan[scanIdx] / HHWeight;
```

Reply:
Hi Karl,

the evtWeight0 is used as weight when calling the EvtHistManager->fillHistograms method. I am missing a term for the lumiScale, data/MC scale factors etc in the line that you proposed. Shouldn't it be
double evtWeight0 = evtWeight * kv.second * Weight_ktScan[scanIdx] / HHWeight;
if you want to add the term kv.second as an additional factor ?

Cheers,

Christian